ns/index.html

```diff
@@ -644,6 +644,14 @@ <h3 id="x4-1-activitypub"><span class="secno">4.1 </span>ActivityPub</h3>
       </ul>
     </section>
 
+    <section id="additional-terms">
+      <h3 id="x4-2-additional-terms"><span class="secno">4.2 </span>Additional terms</h3>
+      <p>The terms in this section are linked to their normative definitions in external specifications.</p>
```

Comment:
Should we make this structure parallel with the way we reference the ActivityPub spec, and have a section like "All terms in this section are described in the [<cite><a class="bibref" href="#bib-did-core">DID Core</a></cite>] specification?

Reply:
I'm glad you said something, I really wasn't sure how to organise this. I thought I'd keep it general because it seems likely there will only ever be one term in DID Core here, so making it a whole section by itself felt like overkill. I also wanted a bit to de-emphasise the idea that `alsoKnownAs` is a DID-specifiic thing, because of course it definitely isn't. But I'm open to a better section heading than "Additional terms"..

Reply:
I don't have a strong preference here, so if you two can reach consensus I'll consider that good enough for me.

Reply:
I don't think it's overkill to have a separate section for each additional normative spec we reference! Since this is mostly an informative document, I think we can restructure later if it turns out to be too heavy-weight and we need to add more specs, but I think, as a person reading this document, a 1:1 relationship of "normative spec docs" and "high-level sections" is what would make the most sense to me. Thoughts?

Reply:
Sure, I'll push another commit to do that in a second then. I don't feel too strongly about it. And you're right, it's editorial such that we could rearrange the sections any time without any kind of vote, so long as the substance remains the same.